src/contexts/AzureMapVectorTileSourceProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import atlas from 'azure-maps-control'
+import React, { useContext, useState } from 'react'
+import { useCheckRef } from '../hooks/useCheckRef'
+import { DataSourceType, IAzureDataSourceStatefulProviderProps, IAzureMapLayerEventType, IAzureMapsContextProps, IAzureMapSourceEventType, IAzureVectorTileSourceStatefulProviderProps, MapType } from '../types'
```

Comment:
`IAzureDataSourceStatefulProviderProps` and  `IAzureMapLayerEventType` are unused, please remove